src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -62,14 +62,16 @@ protected JSRuntime()
         /// <param name="identifier">An identifier for the function to invoke. For example, the value <c>"someScope.someFunction"</c> will invoke the function <c>window.someScope.someFunction</c>.</param>
         /// <param name="args">JSON-serializable arguments.</param>
         /// <returns>An instance of <typeparamref name="TValue"/> obtained by JSON-deserializing the return value.</returns>
-        public ValueTask<TValue> InvokeAsync<TValue>(string identifier, object[] args)
+        public async ValueTask<TValue> InvokeAsync<TValue>(string identifier, object[] args)
         {
             if (DefaultAsyncTimeout.HasValue)
             {
-                return InvokeWithDefaultCancellation<TValue>(identifier, args);
+                using var cts = new CancellationTokenSource(DefaultAsyncTimeout.Value);
+                // We need to await here due to the using
+                return await InvokeAsync<TValue>(identifier, cts.Token, args);
```

Comment:
Change related to nullable or an unrelated issue?

Reply:
It avoids re-asserting the null forgiveness in the other method. Inling it avoids this.